container/docker/handler_test.go

```diff
@@ -169,7 +169,7 @@ func TestAddDiskStatsCheck(t *testing.T) {
 	fileSystems := []fs.Fs{fileSystem}
 
 	var fsStats info.FsStats
-	addDiskStats(fileSystems, nil, &fsStats)
+	fs.AddDiskStats(fileSystems, nil, &fsStats)
```

Comment:
This test should be moved to `fs` module.

Reply:
done~